holoviews/ipython/__init__.py

```diff
@@ -224,19 +224,25 @@ def load_hvjs(cls, logo=False, bokeh_logo=False, mpl_logo=False, plotly_logo=Fal
         import jinja2
         # Evaluate load_notebook.html template with widgetjs code
         if JS:
-            widgetjs, widgetcss = Renderer.html_assets(extras=False, backends=[])
+            widgetjs, widgetcss = Renderer.html_assets(extras=False, backends=[], script=True)
         else:
             widgetjs, widgetcss = '', ''
         templateLoader = jinja2.FileSystemLoader(os.path.dirname(os.path.abspath(__file__)))
         jinjaEnv = jinja2.Environment(loader=templateLoader)
         template = jinjaEnv.get_template('load_notebook.html')
-        display(HTML(template.render({'widgetjs':    widgetjs,
-                                      'widgetcss':   widgetcss,
-                                      'logo':        logo,
-                                      'bokeh_logo':  bokeh_logo,
-                                      'mpl_logo':    mpl_logo,
-                                      'plotly_logo': plotly_logo,
-                                      'message':     message})))
+        html = template.render({'widgetcss':   widgetcss,
+                                'logo':        logo,
+                                'bokeh_logo':  bokeh_logo,
+                                'mpl_logo':    mpl_logo,
+                                'plotly_logo': plotly_logo,
+                                'message':     message})
+        publish_display_data(data={'text/html': html})
+        if JS:
+            publish_display_data(data={
+                MIME_TYPES['js']   : widgetjs,
+                MIME_TYPES['jlab-hv-load'] : widgetjs
+            })
```

Comment:
I'll add a comment to explain that jupyterlab will only process the non-standard mime type.